packages/frontend/src/retro/context/RetroContext.tsx

```diff
@@ -47,6 +48,9 @@ const initialState: RetroState = {
   participants: {},
   waitingList: {},
   isVotingEnabled: false,
+  isTimerPaused: false,
+  isTimerRunning: false,
```

Comment:
I think it makes sense to summarize this state, since there are invalid combinations, e.g. If both values are true. A tiker can not be running or paused at the same time.

There are three states in total: RUNNING, PAUSED, STOPPED.

We could just define an enum here.